### PR TITLE
Added Update to CircleImageView Library

### PR DESCRIPTION
**Pull request by jaimehta19**
## Explanation

Added update to the `CircleImageView` from version `3.0.1` to version `3.1.0` which was released on January 26th, 2020.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml:9]: Revert changes to this file. These are unintentional changes.

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
       </set>
     </option>
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/runConfigurations.xml:13]: Revert changes to this file. These are unintentional changes.

File: app/build.gradle [hunk 3]
```diff
@@ -121,7 +121,7 @@ dependencies {
       'com.google.firebase:firebase-crashlytics:17.0.0',
       'com.google.guava:guava:28.1-android',
       'com.github.oppia:CircularImageview:35d08ba88a',
-      'de.hdodenhof:circleimageview:3.0.1',
+      'de.hdodenhof:circleimageview:3.1.0',
```
Comment (rt4914) [inline app/build.gradle:124]: You will need to apply this change in `third_party/versions.bzl` file too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@jaimehta1912 PTAL Thanks.

**Comment by rt4914** [comment]
@jaimehta1912 Please make sure you have signed the CLA [here](https://github.com/oppia/oppia-android/wiki#onboarding-instructions).

**Comment by jaimehta19** [comment]
@rt4914  yes  sir  ,I have signed CLA already

**Comment by rt4914** [comment]
> @rt4914 yes sir ,I have signed CLA already

@jaimehta1912 Please reply to above comments and update the PR accordingly. Thanks.

**Comment by BenHenning** [comment]
Thanks @jaimehta1912. FYI I'm closing this for the same reason mentioned in https://github.com/oppia/oppia-android/pull/3493#issuecomment-889642775. Similarly, if you think there's a reason why we should update this, then please let us know in a follow-up comment.

**Comment by jaimehta19** [comment]
@BenHenning The reason to Update this particular Library is that new features such as, support for image alpha, replaced fields holding bitmap dimensions with local variables, removed pointless override of getScaleType, restructured initialization and property updates and added support for animated drawables have been released with the Updated version of this Library.

**Comment by jaimehta19** [comment]
@rt4914

**Pull request closed without merging**